### DISCUSSION TITLE
Any milestone supporting jetpack compose :)))

### DISCUSSION THREAD

**Discussion by mochadwi** [ReactiveCircus/FlowBinding#125] [Ideas]
Well, jetpack compose now in beta, any milestone to add extensions for jetpack compose components?

**Comment by mochadwi** [comment]
maybe `implementation "io.github.reactivecircus.flowbinding:flowbinding-compose:${flowbinding_version}"` :)))

**Reply by ychescale9** [reply]
I don't think FlowBinding is needed in Compose where all events are modelled as lambdas. What extensions / APIs do you expect from `flowbinding-compose`?

**Reply by mochadwi** [reply]
On a second thought, I agreed with you, all events in Compose are actually modelled as lambdas, let me get back to the team.

Thanks for the reply!

**Reply by AlexTrotsenko** [reply]
@mochadwi as far as I see, the problem of current compose implementation is that you get an independent calls from the compose, e.g. on multiple button clicks, etc.

It would be great to have a some sort of Flow, where you can apply _filter_ or _debounce_ operators. 
Or am I missing something in compose to achieve this?
